pkg/controller/replication/replication_controller.go

```diff
@@ -242,6 +220,7 @@ func (rm *ReplicationManager) Run(workers int, stopCh <-chan struct{}) {
 	go rm.podController.Run(stopCh)
 	for i := 0; i < workers; i++ {
 		go wait.Until(rm.worker, time.Second, stopCh)
+		go wait.Until(rm.abandoner, 0, stopCh)
```

Comment:
I think that the abandoner should be a part of worker (I'm not yet sure how to do it, just a general level observation, that races are bad)